auth.php

```diff
@@ -266,6 +235,56 @@ public function can_be_manually_set() {
     public function signup_form() {
         return new \auth_enrolkey\form\enrolkey_signup_form(null, null, 'post', '', array('autocomplete' => 'on'));
     }
-}
 
+    /**
+     * @param bool $notify
```

Comment:
Why would you bother with this flag? Why not just not call the method in the first place?

Reply:
true, I'll  update that, I just copied and pasted the existing code :)